core/cloudflow-operator/src/main/scala/cloudflow/operator/action/Actions.scala

```diff
@@ -93,7 +93,20 @@ object Actions {
       currentApp: Option[CloudflowApplication.CR],
       deleteOutdatedTopics: Boolean = false
   )(implicit ctx: DeploymentContext): Seq[Action[ObjectResource]] =
-    TopicActions(newApp, currentApp, deleteOutdatedTopics)
+    if (ctx.kafkaContext.strimziTopicOperatorNamespace.nonEmpty && ctx.kafkaContext.strimziClusterName.nonEmpty) {
```

Comment:
Shouldn't this be a `useStrimzi` method on `KafkaContext`?

Reply:
yeah could do that. This is really a quick fix..

Reply:
Done.